ui/src/mixins/virtual-scroll.js

```diff
@@ -460,7 +460,7 @@ export default {
             h('tr', [
               h('td', {
                 style: { [paddingSize]: `${this.virtualScrollPaddingAfter}px` },
-                attrs: { colspan: '100%' }
+                attrs: { colspan: this.columnCount || '100%' }
```

Comment:
change this to attrs: { colspan: this.columnCount === void 0 ? '100%' : this.columnCount }